app/api/entities/metadataValidators.js

```diff
@@ -73,8 +73,23 @@ const validateGeolocationProperty = value =>
 const validateMultiSelectProperty = value =>
   Array.isArray(value) && value.every(item => isValidSelect(item.value) && item.value);
 
+const validateLuceneBytesLimit = (property, value) => {
+  let valid = true;
+  if (
+    !property ||
+    property.type === propertyTypes.text ||
+    property.type === propertyTypes.markdown
+  ) {
+    const LUCENE_BYTES_LIMIT = 32766;
+    const bytes = Buffer.from(JSON.stringify(value));
+    valid = bytes.length < LUCENE_BYTES_LIMIT;
+  }
+  return valid;
+};
+
```

Comment:
Why do we need the property type check?  Does the Lucene limit not apply to any field? What happens if someone inputs a Numeric field that exceeds this?  I know it's absurd, no one will do that, but I am concerned with the maintainability of this piece of code: if we add other types of fields, we have to always remember and add them here... it's the sort of thing that is prone to development error / forgetting.

Could this not be just a global rule? cc. @daneryl

Reply:
I think that for other fields its own validations of format or type prevent to admit long fields and in that case, we could avoid this processing block

Reply:
looks good now 👍